CRM/Upgrade/Incremental/sql/4.7.14.mysql.tpl

```diff
@@ -14,3 +14,9 @@ UPDATE civicrm_navigation SET
   `url` = 'civicrm/tag?reset=1'
 WHERE `name` = 'Tags (Categories)'
 AND `url` = 'civicrm/admin/tag?reset=1';
+
+--  CRM-19517 Disable all price fields nad price field options that use disabled fianncial types
+UPDATE civicrm_price_field_value cpfv
+INNER JOIN civicrm_financial_type cft oN cft.id = cpfv.financial_type_id
+SET cpfv.is_active = 0
+WHERE cft.is_active = 0;
```

Comment:
@JoeMurray  should have a solution for that shortly, and with your other comment if your meaning this https://github.com/civicrm/civicrm-core/pull/9276/files#diff-2bd325d7967360f15458894201f5cba9R128 then all that is doing is saying are we getting a financial type passed through otherwise load it from the current record (in case someone is just clicking enable where its doing an API call in without passing in any new financial type info)